From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 30 Mar 2014 01:54:10 +0100
Subject: Install data files too
License: GPL-3+

---
 setup.py |   11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

Index: caldav-tester-packaging/setup.py
===================================================================
--- caldav-tester-packaging.orig/setup.py	2016-02-02 09:56:11.659193948 +0100
+++ caldav-tester-packaging/setup.py	2016-02-02 09:56:30.571371957 +0100
@@ -19,6 +19,7 @@
 from __future__ import print_function
 
 from os.path import dirname, abspath, join as joinpath
+from os import walk
 from setuptools import setup, find_packages as setuptools_find_packages
 import errno
 import os
@@ -41,6 +42,13 @@
         ])
     return modules
 
+def find_data():
+    data = []
+
+    for d in [ 'Resource', 'scripts' ]:
+        for root, dirs, files in walk(d):
+            data += [("share/caldavtester/"+root, [ joinpath(root, file) for file in files ])]
+    return data
 
 
 def version():
@@ -173,7 +181,7 @@
         packages=find_packages(),
         package_data={},
         scripts=[],
-        data_files=[],
+        data_files=find_data(),
         ext_modules=extensions,
         py_modules=[],
         setup_requires=setup_requirements,
