From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 30 Mar 2014 01:54:10 +0100
Subject: Install data files too

---
 setup.py |   11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index a3bad13..c3fd46c 100755
--- a/setup.py
+++ b/setup.py
@@ -19,7 +19,7 @@
 from __future__ import print_function
 
 from itertools import chain
-from os import listdir, environ as environment
+from os import walk, listdir, environ as environment
 from os.path import dirname, abspath, join as joinpath
 import errno
 import subprocess
@@ -36,6 +36,13 @@ def find_packages():
 
     return modules + setuptools_find_packages()
 
+def find_data():
+    data = []
+
+    for d in [ 'Resource', 'scripts' ]:
+        for root, dirs, files in walk(d):
+            data += [("share/caldavtester/"+root, [ joinpath(root, file) for file in files ])]
+    return data
 
 
 def version():
@@ -179,7 +186,7 @@ def doSetup():
         packages=find_packages(),
         package_data={},
         scripts=[],
-        data_files=[],
+        data_files=find_data(),
         ext_modules=extensions,
         py_modules=[],
         setup_requires=setup_requirements,
